plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreMessages.java

```diff
@@ -53,6 +53,9 @@ public class CoreMessages extends NLS {
     public static String action_menu_transaction_auto_close_enabled;
     public static String action_menu_transaction_auto_close_enabled_tip;
     public static String action_menu_transaction_auto_close_ttl_tip;
```

Comment:
We already have this localization (dialog_connection_wizard_final_label_close_idle_connections)

Reply:
These dedicated prefereces page localization
![image](https://github.com/dbeaver/dbeaver/assets/32031248/2d8c4947-29f5-4b81-91ad-666ae2be123b)